package.json

```diff
@@ -5,21 +5,23 @@
   "scripts": {
     "build": "npm run clean && mkdirp build && webpack --progress --colors && webpack --config=demos/webpack.config.js --progress --colors",
     "build:min": "mkdirp build && cross-env MDC_ENV=production webpack -p --progress --colors",
+    "build:screenshot": "npm run clean && webpack --config=test/screenshot/webpack.config.js --progress --colors",
```

Comment:
I'll remove `build:screenshot` until we have a real-world need for it.
I was trying to think too far ahead 🙂 

`dev:screenshot` needs `MDC_STATIC_SERVER=true webpack` because it's calling a single shared config file that needs to support multiple "feature flags" (e.g., "start up the dev server" vs. "upload screenshot assets to GCS").

`build:screenshot` didn't need the flag because it wasn't starting up a local dev server - it was only compiling and outputting the static assets for it, which _always_ happens when you run `webpack.config.js`.